SwiftVoiceCallKitQuickstart/ViewController.swift

```diff
@@ -423,7 +425,6 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
 
             // RCP: Workaround per https://forums.developer.apple.com/message/169511
             TwilioVoice.configureAudioSession()
-            TwilioVoice.isAudioEnabled = false
```

Comment:
I'm honestly curious, is this block called first or the provider delegate method? I still agree that this change makes sense.